### PR TITLE
Cover 1266842 BZ functionality

### LINKED ISSUE #3301: Automate scenario for BZ 1266842
The main idea here is to delete existing docker container using API and then verify that it was really deleted through WebUI.

Also, it will be really nice to cover same test case using CLI instead API

### PR DESCRIPTION
**Pull request by oshtaier**
Closes #3301

### CODE REVIEW

File: tests/foreman/ui/test_docker.py [hunk 1]
```diff
@@ -1279,6 +1280,34 @@ def test_positive_delete(self):
                     self.assertIsNone(
                         self.container.search(compute_resource.name, name))
 
+    # BZ 1266842 is a private bug, so remove that stub once it is fixed
+    @stubbed
```
Comment (abalakh) [inline tests/foreman/ui/test_docker.py]: Maybe it's better to specify a reason here like we did it in [#3277](https://github.com/SatelliteQE/robottelo/pull/3277/files#diff-7348c3a9bfd682d700bc0c8540410544R324) and [#3274](https://github.com/SatelliteQE/robottelo/pull/3274/files#diff-aa8831ff56496154fb9a4a2754c259fdR543)?

**Comment by abalakh** [comment]
ACK pending suggestion

**Comment by oshtaier** [comment]
Addressed

File: tests/foreman/ui/test_docker.py [hunk 2]
```diff
@@ -1279,6 +1280,34 @@ def test_positive_delete(self):
                     self.assertIsNone(
                         self.container.search(compute_resource.name, name))
 
+    # BZ 1266842 is a private bug, so we cannot fetch information about it
+    @stubbed('Unstub when BZ1266842 is fixed')
```
Comment (elyezer) [inline tests/foreman/ui/test_docker.py]: The bad thing about doing this is it's not gonna be included on the skip report due to BZ. To fix this we can add a user and password for BZ in order to fetch private information. But as you may be already thinking, this is a future improvement that should be addressed later.

**Comment by elyezer** [comment]
ACK

File: tests/foreman/ui/test_docker.py [hunk 3]
```diff
@@ -1279,6 +1280,34 @@ def test_positive_delete(self):
...
+    @run_only_on('sat')
+    @tier2
+    def test_verify_bugzilla_1266842(self):
```
Comment (elyezer) [inline tests/foreman/ui/test_docker.py]: I think we should use a meaningful name. As I commented here https://github.com/SatelliteQE/robottelo/pull/3304#discussion_r53657092, eventually the BZ skip will be removed and having a test that requires checking the docstring or the related BZ is not good as one with a meaningful name that by just looking on jenkins you can get information about the test.

**Comment by elyezer** [comment]
ACK, pending suggestion.

File: tests/foreman/ui/test_docker.py (hunk 3 above)
Comment (sthirugn) [inline tests/foreman/ui/test_docker.py]: :+1:

File: tests/foreman/ui/test_docker.py (hunk 2 above)
Comment (oshtaier) [inline tests/foreman/ui/test_docker.py]: Yeah, we have bunch of test cases that are using that approach which is definitely not good and should be improved

**Comment by oshtaier** [comment]
Addressed comments

**Comment by elyezer** [comment]
ACK

**Pull request merged**